sdk/src/main/java/co/omise/android/models/Capability.kt

```diff
@@ -0,0 +1,62 @@
+package co.omise.android.models
+
+import android.os.Parcel
+import android.os.Parcelable
+import co.omise.android.api.Endpoint
+import co.omise.android.api.RequestBuilder
+import okhttp3.HttpUrl
+import org.json.JSONObject
+
+/**
+ * Represents Capabilities object and contains its {@link RequestBuilder}.
+ *
+ * @see <a href="https://www.omise.co/capability-api">Capabilities API</a>
+ */
+class Capability : Model {
+    @JvmField
+    var banks: List<String>? = null
+    @JvmField
+    var paymentMethods: List<PaymentMethod>? = null
+    @JvmField
+    var zeroInterestInstallments: Boolean = false
+
+
+    constructor(rawJson: String) : this(JSONObject(rawJson))
+
+    private constructor(json: JSONObject) : super(json) {
```

Comment:
I think we should use any tools (i.e. Jackson, Gson) to help us parsing Json. However, we should another task to do the serializer.

Reply:
As you also know, this is not my preferred method of parsing and this was already in place since 2016. This [task](https://phabricator.omise.co/T13595) is already in place for us to discuss and choose a suitable serializer library 👍

Reply:
🆗